### PR TITLE
Configure queue attributes before worker start up

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Not able to test it without AWS credentials. Let's see if it works on staging.

File: hack/queue_conf.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        queue_url = response.get('QueueUrl')
+        if not queue_url:
+            raise RuntimeError("Response from remote does not contain QueueUrl, "
+                               "cannot adjust queue attributes, response: {}".format(response))
+
+        response = client.set_queue_attributes(
+            QueueUrl=queue_url,
+            Attributes={
+                'MessageRetentionPeriod': int(timedelta(days=14).total_seconds())
```
Comment (jpopelka) [inline hack/queue_conf.py]: Shouldn't it be configurable ?

Comment (fridex) [reply hack/queue_conf.py]: The way how core works, we will need to make sure AWS retains messages as long as possible.

File: hack/queue_conf.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+if __name__ == '__main__':
+    if os.getenv('F8A_UNCLOUDED_MODE') in ('1', 'True', 'true'):
+        _logger.warning("Worker started in unclouded mode, "
+                        "no queue configuration adjustment will be performed")
+        sys.exit(0)
+
+    if sys.argv[1] in ('-h', '--help') or len(sys.argv) != 2:
+        print_help()
+        sys.exit(1)
```
[the remark this answers is not in the document]
Comment (fridex) [reply hack/queue_conf.py]: Fixed in newer version bbc2b2e

**Pull request merged**